catalog/imaging/catalog-video-transformer/src/main/resources/videoMetcardType.json

```diff
@@ -0,0 +1,286 @@
+{
+  /*
```

Comment:
I don't think comments are allowed in JSON files. Also, there's a typo in the file title: `videoMetcardType`.